website/static/js/institutionProjectSettings.js

```diff
@@ -13,16 +12,15 @@ var ViewModel = function(data) {
     self.loading = ko.observable(true);
     self.showAdd = ko.observable(false);
     self.institutionHref = ko.observable('');
-    self.userInstitutions = window.contextVars.currentUser.institutions;
-    var userInstitutionsIds = self.userInstitutions.map(function (item) {
-        return item.id;
-    });
-    self.userInstitutionsIds = ko.observable(userInstitutionsIds);
+    self.userInstitutions = data.currentUser.institutions;
+    self.userInstitutionsIds = self.userInstitutions.map(function(item){return item.id;});
     self.selectedInstitution = ko.observable();
-    self.affiliatedInstitutions = ko.observable(window.contextVars.node.institutions);
-    self.affiliatedInstitutionsIds = self.affiliatedInstitutions().map(function (item) {
-        return item.id;
-    });
+    self.affiliatedInstitutions = ko.observable(data.node.institutions);
+
+    self.affiliatedInstitutionsIds = self.affiliatedInstitutions().map(function(item){return item.id;});
```

Comment:
Does `self.affiliatedInstitutions` ever change? If it does, then I think affiliatedInstitutionIds will need to be computed.